ext/src/ruby_api/pooling_allocation_config.rs

```diff
@@ -100,16 +100,6 @@ impl PoolingAllocationConfig {
         Ok(rb_self)
     }
 
-    /// @def async_stack_zeroing=(enable)
-    /// @param enable [Boolean]
```

Comment:
Let's move the documentation to `Engine.new` (engine.rs)?

Reply:
Good catch, documented.